client-js/app/elements/app-link.html

```diff
@@ -10,7 +10,11 @@
     },
 
     onClick: function(e) {
-      this.fire('app-link-click', {path: this.path});
+      if (e.ctrlKey || e.metaKey) {
+        return true;
+      } else { 
+        this.fire('app-link-click', {path: this.path});
+      }
```

Comment:
I would suggest just leaving out the else clause since the if clause returns. This makes it clear that preventDefault goes together with the custom event, in response to clicks with no special keys.
```javascript
if (e.ctrlKey || e.metaKey) {
  return true;
}
this.fire('app-link-click', {path: this.path});
e.preventDefault();
```